### PR TITLE
support real multiarch build.

### PR DESCRIPTION
**Pull request by morvencao**
**What this PR does / why we need it**:

resolve: IBM/operand-deployment-lifecycle-manager#241
depends on: https://github.com/IBM/test-infra/pull/146
**We have add power and s390x cluster to the prow, now we can support real multi-arch build, this PR is to replace the old QEMU simulation way.**

/hold
/dot-not-merge
until the stakeholder approve this to avoid build CI jobs down.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes  https://github.com/IBM/operand-deployment-lifecycle-manager/issues/241

**Special notes for your reviewer**:

1. Please confirm that if this PR changes any image versions, then that's the sole change this PR makes.

**Release note**:

```release-note

```

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -121,44 +128,25 @@ endif
...
-	docker build -t $(REGISTRY)/$(IMG)-s390x:$(VERSION) -f build/Dockerfile.s390x .
-	@\rm -f build/_output/bin/odlm-s390x
-	@if [ $(BUILD_LOCALLY) -ne 1 ]; then docker push $(REGISTRY)/$(IMG)-s390x:$(VERSION); fi
-endif
-endif
+build-push-image: build-image push-image
+
+build-image: build
+	@echo "Building the $(IMAGE_NAME) docker image for $(LOCAL_ARCH)..."
+	@docker build -t $(IMAGE_REPO)/$(IMAGE_NAME)-$(LOCAL_ARCH):$(VERSION) -f build/Dockerfile .
```
Comment (horis233) [inline Makefile]: ODLM has three docker files `Dockerfile` `Dockerfile.ppc64le` and `Dockerfile.s390x` for each arch. Their base images are different.

Comment (morvencao) [reply Makefile]: Nice Catch!
Have remove default `Dockerfile` to `Dockerfile.amd64` and add `suffix` for `Dockerfile.$(LOCAL_ARCH)`

**Comment by gyliu513** [comment]
/lgtm

Leave to @chenzhiwei @horis233 @DanielXLee for approve.

**Comment by DanielXLee** [comment]
@morvencao uses the real arch node to build ppc64le and s390x,  is it means we can delete `qemu-ppc64le-static` and `qemu-s390x-static` from dockerfile?

**Comment by morvencao** [comment]
@DanielXLee Oh, yes, I forget that.

**Comment by morvencao** [comment]
Emmmm... the power cluster is slower.

**Comment by morvencao** [comment]
Update dockerfiles, PTAL @odlm-maintainers

**Comment by morvencao** [comment]
Now we use one dockerfile for all three platforms, since the base image is also multiarch image.

**Pull request merged**